src/tests/integration-test

```diff
@@ -246,7 +251,8 @@ class UDisksTestCase(unittest.TestCase):
         If cd is True, return the CD device, otherwise the hard disk device.
         """
         block = klass.udisks_block(partition, cd)
-        return klass.client.get_object(block.get_object_path()).get_filesystem()
+        return klass.client.get_object(
+            block.get_object_path()).get_filesystem()
```

Comment:
Sure, but all those tools have ways to disable specific checks/errors/warnings. PEP is not a standard, the PEP 8 is titled as *Style Guide for Python Code*. Good, a nice guide. But that doesn't mean we have to do exactly as it says. Don't take me wrong, I was the one who started pushing PEP 8 and PEP 257 in Anaconda and Blivet some time ago I know they make things much better. But the 7**2**-characters-per-line is just nonsense these days. And the guide even mentions this:

> Some teams strongly prefer a longer line length. For code maintained exclusively or primarily by a team that can reach agreement on this issue, it is okay to increase the nominal line length from 80 to 100 characters (effectively increasing the maximum length to 99 characters), provided that comments and docstrings are still wrapped at 72 characters.

As for tests being as good as the code -- sure, let's start with the code then, it's much worse than our tests.